android/app/src/main/kotlin/org/getlantern/lantern/model/InAppBilling.kt

```diff
@@ -54,6 +56,15 @@ class InAppBilling(
     @Volatile
     private var purchasesUpdated: PurchasesUpdatedListener? = null
 
+
+    fun isPlayStoreAvailable(): Boolean {
+        if (billingResult != null) {
```

Comment:
Beyond checking the billing client is initialized, does it make sense to use something like [GoogleApiAvailability.isGooglePlayServicesAvailable()](https://developers.google.com/android/reference/com/google/android/gms/common/GoogleApiAvailability#isGooglePlayServicesAvailable%28android.content.Context%29) here to check that Google Play services is installed and enabled on the device?

Reply:
According to this-:https://stackoverflow.com/questions/52858403/check-if-in-app-billing-via-playstore-is-available-to-user, billingResult will work just fine.

Reply:
Sounds good!